iroha/src/smartcontracts.rs

```diff
@@ -0,0 +1,52 @@
+//! Module that opts different smartcontract runtime
```

Comment:
Better to name it `runtime` in my opinion.

Reply:
Traits are not part of runtime, so may be it would better to call it just `smartcontracts`.

Reply:
To my mind traits - Execute and Evaluate - are exactly the parts of runtime. Smart contracts are usually something that users of blockchain write. Here we basically have a smart contract runtime, interpreter etc.

Reply:
Traits themselves are part of runtime specification not runtime itself. It is like language spec and compiler.

Reply:
Well yes, everything that is inside there is a part of runtime spec, so semantically they all should be in `runtime/isi`

Reply:
But I am not sure if such deep nesting is needed now